src/storybook/get-stories/extract-stories/v3.ts

```diff
@@ -0,0 +1,17 @@
+import type { StorybookRawStory } from "./index";
+import { STORYBOOK_STORIES_JSON_PATH } from "../../../constants";
+
+export const supportedStoryVersionV3 = 3 as const;
+
+export type StorybookStoriesJson = {
+    v: typeof supportedStoryVersionV3;
+    stories: Record<string, Omit<StorybookRawStory, "type">>;
+};
+
+export const storybookDataJsonPathV3 = STORYBOOK_STORIES_JSON_PATH;
+
+export const extractStoriesV3 = (dataJson: StorybookStoriesJson): StorybookRawStory[] => {
+    return Object.values(dataJson.stories)
+        .filter(story => !story.tags?.includes("docs"))
```

Comment:
if user enables `buildStoriesJson` on `v7`, docs components would appear in those list, so we filter those out.